README.md

```diff
@@ -6,7 +6,6 @@
 [![angular-open-source-starter](https://img.shields.io/badge/made%20with-angular--open--source--starter-d81676?logo=angular)](https://github.com/tinkoff/angular-open-source-starter)
 
 [Website](https://taiga-ui.dev) • [Documentation](https://taiga-ui.dev/getting-started) •
-[CDK Wiki](https://github.com/tinkoff/taiga-ui/wiki) • [Core team](#core-team) •
```

Comment:
why do you remove `[Core team](#core-team)` ?

btw, add yourself to core team too :)

Reply:
and here too
https://github.com/Tinkoff/taiga-ui/blob/bb6d824c37a986d248e7af1aa1f2c7aa9c13a7b1/.github/CODEOWNERS#L13

Reply:
fixed)